src/app/sidenav/sidenav-link/sidenav-link.component.view.spec.ts

```diff
@@ -41,6 +41,28 @@ describe('MzSidenavLinkComponent:view', () => {
       });
     }));
 
+    it('should have active class when active prop is provided', async(() => {
+
+      buildComponent<MzSidenavLinkComponent>(`<mz-sidenav-link [active]="true"></mz-sidenav-link>`).then((fixture) => {
+
+        nativeElement = fixture.nativeElement;
+        fixture.detectChanges();
+
+        expect(sidenavLink().classList).toContain('active');
```

Comment:
Your "active" class is applied on `<li>` element. The function `sidenavLink` return the `<mz-sidenav-link>` element.
You should look at the previous test on how we get the `<li`>` element.